### ISSUE TITLE
get_account does not allow you to set a region

### ISSUE THREAD

**Issue by lbkulinski** [meraki-analytics/cassiopeia#452]
Hello.

I am experimenting with Cassiopeia and found a potential bug with `get_account`:
```
import os

import cassiopeia as cass
from fastapi import FastAPI

api_key = os.environ.get('RIOT_API_KEY')

cass.set_riot_api_key(api_key)

app = FastAPI()


@app.get("/api/lol/me")
async def root():
    account = cass.get_account(name='lbku', tagline='lbku')

    return {
        "account": account.to_dict()
    }
```

When I hit the endpoint, I get the following error:
```
TypeError: __get_query_from_kwargs__() missing 1 required keyword-only argument: 'region'
```

If I try to specify the region via `cass.get_account(name='lbku', tagline='lbku', region='NA')`, I also get an error:
```
TypeError: get_account() got an unexpected keyword argument 'region'
```

Thanks,
Logan

**Comment by 66Omar** [comment]
Hello, @jjmaldonis,
Your last commit says WIP, are you currently working on this update? I managed to fix this locally but still haven't tested the rest of the functionality, if there are no pending changes on your side, I can create a PR hopefully by the end of the day

**Comment by rraygoza99** [comment]
Hi! Im just getting this same error:
```
def get_summoner_info():
    if 'region' not in request.args or 'game_name' not in request.args or 'tag_line' not in request.args:
        return jsonify({"error": "Missing required parameters: 'region', 'game_name', and 'tag_line'"}), 400
    region = request.args['region']
    game_name = request.args['game_name']
    tag_line = request.args['tag_line']
    try:
        account = cass.get_account(name=game_name, tagline=tag_line, region=region)
        return jsonify(account.summoner)
    except Exception as e:
        return jsonify({"error": str(e)}), 500
```

If I remove the region parameter I get: "**TypeError: __get_query_from_kwargs__() missing 1 required keyword-only argument: 'region'**"

Thanks for the support!! :D

**Comment by jjmaldonis** [comment]
This is fixed by #453 and I made a new release on PyPi.
